ui/js/cache-templates.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+// This is a run block which, when used on an angular module, loads all
+// of the partials in /ui/partials and /ui/plugins into the Angular
+// template cache. This means that ng-includes and templateUrls will not
+// actually need to request a partial file at any point.
```

Comment:
I attempted to use ngtemplate-loader and ng-cache-loader before trying this approach - they worked in cases where the template was included via `templateUrl: require('....html')`, but I couldn't make them work with `ng-include` in the main templates (index.html etc.) which are rendered by the html plugin. It may be possible, but I just couldn't find a way to make them work together successfully.